core/src/main/java/org/infinispan/marshall/core/ExternalizerTable.java

```diff
@@ -384,6 +385,8 @@ private void loadInternalMarshallables() {
       addInternalExternalizer(new MarshallableFunctionExternalizers.LambdaWithMetasExternalizer());
       addInternalExternalizer(new MarshallableFunctionExternalizers.SetValueIfEqualsReturnBooleanExternalizer());
       addInternalExternalizer(new PersistentUUID.Externalizer());
+
+      addInternalExternalizer(Params.Externalizer.INSTANCE);
```

Comment:
If you always call the externalizer's methods directly, this registration will never be used. So you could remove the `Params.Externalizer` class altogether and make the methods static in `Params`.